docs/shared/guides/remix.md

```diff
@@ -0,0 +1,181 @@
+# Remix with Nx
+
+In this recipe, we'll show you how to create a [Remix](https://remix.run) application with Nx.
+
+## Create Nx Workspace
+
+```{% command="npx create-nx-workspace acme --preset=apps" path="~/" %}
```

Comment:
should we show both the adding to a workspace + using the remix preset? 
```shell
npx create-nx-workspace@latest acme \
--preset=@nx/remix \
--project=demo
```

Reply:
I'm going to stick to just adding to a workspace for now as we have plans to move Remix into the main repo and have it's own dedicated CNW flow under the React stacks